ibis/expr/operations/arrays.py

```diff
@@ -135,3 +135,54 @@ def output_dtype(self):
         return self.arg.output_dtype.value_type
 
     output_shape = rlz.Shape.COLUMNAR
+
+
+@public
+class ArrayContains(Value):
+    arg = rlz.array
+    other = rlz.any
+
+    output_dtype = dt.boolean
+    output_shape = rlz.shape_like("args")
+
+
+@public
+class ArrayPosition(Value):
```

Comment:
Shall we rename ArrayIndex to ArreyGet and use ArrayIndex here instead? That would better align with the python semantics and the method naming we use on the expr level